app/javascript/packs/user.js

```diff
@@ -0,0 +1,89 @@
+$(document).ready(function(){
+  $.ajaxSetup({
+    headers: {
+      'X-CSRF-Token': $('meta[name="csrf-token"]').attr('content')
+    }
+  });
+
+  $('.delete-item-cart').click(function(){
+    let parent = $(this).parents('li');
```

Comment:
đặt tên có dấu $ trước để phân biệt biến select element với biến giá trị
```suggestion
    let $parent = $(this).parents('li');
```

Reply:
master FE sugggest mà ko fix bợm, chém chừ

Reply:
Dạ trước khi ready thì 2 anh em cũng đã nch rồi chị. 
Cả 2 cũng chưa chắc nên ready cho chị review rồi sửa tiếp ạ